internal/newsfeed/new_feed_manager_test.go

```diff
@@ -0,0 +1,71 @@
+package newsfeed
+
+import (
+	"testing"
+	"time"
+
+	"github.com/mmcdole/gofeed"
+	"github.com/stretchr/testify/suite"
+)
+
+type NewsFeedManagerSuite struct {
+	suite.Suite
+}
+
+type MockParser struct {
+	Feed *gofeed.Feed
+	Err  error
+}
+
+func (mp *MockParser) ParseURL(url string) (*gofeed.Feed, error) {
+	return mp.Feed, mp.Err
+}
+
+type MockHandler struct {
+	callback func(item *gofeed.Item)
+}
+
+func (mh *MockHandler) HandleFeed(item *gofeed.Item) {
+	mh.callback(item)
+}
```

Comment:
It's fine in this case (as it's a small one), but in the future I really encourage you to use [`gomock`](https://github.com/uber-go/mock). There're many examples in our codebase 🙌